package.json

```diff
@@ -180,6 +180,8 @@
         "leaflet": "^1.7.1",
         "lodash-es": "^4.17.20",
         "mini-svg-data-uri": "^1.4.3",
+        "monaco-editor": "^0.33.0",
+        "monaco-editor-webpack-plugin": "^7.0.1",
```

Comment:
```suggestion
```

This library is not needed when you use monaco-react because monaco-editor on CDN will be used

https://github.com/suren-atoyan/monaco-react/issues/65#issuecomment-602836571